.github/workflows/test-e2e-cron.yml

```diff
@@ -0,0 +1,38 @@
+name: E2E tests against most recent Polkadot release
+on:
+  schedule:
+    - cron: 0 3 * * SUN # Once a week on Sunday.
+  workflow_dispatch:
+
+jobs:
+  test-e2e-cron:
+    runs-on: ubuntu-22.04
+    timeout-minutes: 120
+    container: paritytech/ci-linux:production
```

Comment:
unified?
i guess the production isn't supported anymore

Reply:
Ah right, I missed the second workflow - updated.